cybersyn/settings.lua

```diff
@@ -67,7 +67,7 @@ data:extend({
 		order = "be",
 		setting_type = "runtime-global",
 		default_value = .5,
-		minimum_value = 0,
+		minimum_value = -1,
```

Comment:
This probably deserves some explanation in a tooltip because it's not intuitive. Also, this a floating point value, what happens for values >-1 and <0?

Reply:
Nothing extraordinary. These values behave the same as -1.

We just need something less than zero, so anything below zero works. -1 was a nice rounded number.

I thought about adding a separate checkbox or adjusting the behaviour when user selected zero, but having negative amount achieved the same with backwards compatibility and no extra settings.